libs/libarchfpga/src/device_grid.cpp

```diff
@@ -18,19 +18,15 @@ DeviceGrid::DeviceGrid(std::string grid_name,
 size_t DeviceGrid::num_instances(t_physical_tile_type_ptr type, int layer_num) const {
     size_t count = 0;
     //instance_counts_ is not initialized
-    if (instance_counts_.empty()) {
-        return 0;
-    }
+    if (instance_counts_.empty()) { return 0; }
```

Comment:
AllowShortIfStatementsOnASingleLine: true

Reply:
I don't like single line ifs -- turning it off makes sense to me.

Reply:
I agree. Please turn this off. I would prefer if the above were turned into:
```cpp
if (instance_counts_.empty())
    return 0;
```
The curly braces are not necessary.